### PR TITLE
fix(wallet): getStore Updates

### PR DESCRIPTION
**Pull request by coreyphillips**
This PR:
- Reduces usage of `getStore` where able.
- Splits `getStore` into several methods for individual stores.

### CODE REVIEW

File: src/store/helpers.ts [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+import { IUser } from './types/user';
 
 /*
 Used to retrieve the store outside of a component.
  */
 export const getStore = (): Store =>
 	JSON.parse(JSON.stringify(store.getState()));
 
+export const getWalletStore = (): IWallet =>
+	JSON.parse(JSON.stringify(store.getState()[EStore.wallet]));
```
Comment (pwltr) [inline src/store/helpers.ts:21]: Is there a reason to first stringify and then parse in the same line here? Usually can just access `store.getState()`

Comment (coreyphillips) [reply src/store/helpers.ts:21]: Prevents mutating state by writing directly to the store.

Comment (pwltr) [reply src/store/helpers.ts:21]: Seems an expensive way to clone an object, or do you think it's optimized enough by the JS engine?

Comment (coreyphillips) [reply src/store/helpers.ts:21]: It's optimized enough for now, but yes I agree, we'll eventually need a better solution. Previous attempts to try something else yielded odd/inconsistent results.

**Review by pwltr — APPROVED** [review]

**Pull request merged**